check_commits.sh

```diff
@@ -90,15 +90,15 @@ do
         continue
     fi
 
-    if [ ! -z $CHECK_SIGNOFFS ]; then
+    if [ -n ${CHECK_SIGNOFFS} ]; then
```

Comment:
Nitpick :straight_ruler: : 
Dirtying blame log

Reply:
Hehe :policeman: 
I can argue that this is related: I did modify `! -z` to `-n` in the other line to make it more read-able (as now it it fi this and that); and then my consistency check instinct told me to apply it here also.